packages/nextjs/test/config.test.ts

```diff
@@ -34,6 +35,10 @@ const userNextConfig = {
 };
 const userSentryWebpackPluginConfig = { org: 'squirrelChasers', project: 'simulator', include: './thirdPartyMaps' };
 
+/** mocks of the arguments passed to the result of `withSentryConfig` (when it's a function) */
+const runtimePhase = 'puppy-phase-chew-everything-in-sight';
```

Comment:
nit: Can we use a phase from `next/constants` that better reflects what this would be called with in real usage? As per phase docs in https://nextjs.org/docs/api-reference/next.config.js/introduction

Reply:
¯\\_(ツ)_/¯ We can, if you think it's important. Since they're values which are merely passed from our function to the user's function, they can really be anything. I find writing (and reading) tests to be kinda uninteresting, so I try to have some fun with it.

Regardless, I'm going to merge this now, in the interest of moving on.